README.md

```diff
@@ -13,6 +13,8 @@
 
 # Configuring Your Local Environment
 
+You can deploy the stack from the following repository: [AWS VPC Creator](https://github.com/SofarDatas/aws-vpc-creator). After deployment, you can use the VPC_ID obtained from there as an input parameter.
```

Comment:
**Suggestion:** Provide a direct link to the AWS VPC Creator repository for easier access to obtain the `VPC_ID`.
> 🚵  **Medium Priority** 
 > [category: **`enhancement`**, importance: **`6`**]
> **Reason:** The suggestion improves the readability of the documentation by providing a direct link to the AWS VPC Creator repository, making it easier for users to access the resource.
 

```suggestion
16 +You can deploy the stack from the [AWS VPC Creator](https://github.com/SofarDatas/aws-vpc-creator) repository. After deployment, you can use the VPC_ID obtained from there as an input parameter.
```